drivers/i2c/i2c_mchp_xec.c

```diff
@@ -493,53 +774,127 @@ static void i2c_xec_bus_isr(void *arg)
 		if (slave_cb->stop) {
 			slave_cb->stop(data->slave_cfg);
 		}
-		dummy = MCHP_I2C_SMB_DATA(ba);
+		/* dummy = MCHP_I2C_SMB_DATA(ba); */
```

Comment:
The more I work with this I2C controller the less I trust the data sheet. The data sheet states there are two ways to "clean up" after receiving the externally generated STOP.  Dummy read of I2C Data register (original code) OR write PIN=1 in I2C Control.  Since I2C Control must be written to 0xC1 setting PIN=1 then why do both?   I will remove the commented out line.